tests/conftest.py

```diff
@@ -46,3 +55,28 @@ def pytest_ignore_collect(
         return not cast(bool, config.getoption("integration"))
 
     return False
+
+
+def pytest_runtest_setup(item: pytest.Function) -> None:
+    marker = item.get_closest_marker(name="fairseq2n")
+    if marker is not None:
+        skip_if_fairseq2n_newer(*marker.args, **marker.kwargs)
```

Comment:
if we end up using custom marks can we have a link to documentation ? https://docs.pytest.org/en/stable/reference/reference.html#custom-marks

Reply:
Yep. I just realized that I should also update the contribution guidelines and mention this annotation there.